packages/dtslint/src/lint.ts

```diff
@@ -27,8 +27,8 @@ export async function lint(
   // TODO: To remove tslint, replace this with a ts.createProgram (probably)
   const lintProgram = Linter.createProgram(tsconfigPath);
```

Comment:
We only use this program to build a list of files; not for this PR, but we're going to have to remove this when we remove tslint so I expect that we will just ask eslint to run on all non-ignored files (which is a lot better than having to find a list of all files here).

Reply:
That being said, since we run type-using rules and also use project=true, it's likely that people will have random files that _aren't_ in the program that may break via ts-eslint complaining about not being in a program, or via new lints that were previously uncaught.

Reply:
Cost reduction: `Linter.createProgram` is a standalone `public static`, so an equivalent could always be re-written.

Reply:
Probably, yeah, though it seems a bit overkill to build a Program to only use it to list files!